sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc

```diff
@@ -472,10 +508,54 @@ architecture.
 |-
 |GCN GFX10.3 (RDNA 2) architecture.
 
+|`amd_gpu_gfx1033`
+|-
+|GCN GFX10.3 (RDNA 2) architecture.
```

Comment:
These new entries are described as "GCN" while the previous ones are described as "AMD GCN".  Is that an important difference, or is it unintentional?

Reply:
I will add "AMD". Thanks.